all_test.go

```diff
@@ -17,12 +17,15 @@ import (
 //go:embed test/data/case18/assets
 var assets embed.FS
 
+var supportsWrapReaderOption = true
+
 func setupFileCopyMethod(m *testing.M) {
 	// Allow running all the tests with a different FileCopyMethod.
 	// We want to be able to have full coverage no matter the method.
 	switch os.Getenv("TEST_FILECOPYMETHOD") {
 	case "CopyBytes":
 		defaultCopyMethod = CopyBytes
+		supportsWrapReaderOption = true
```

Comment:
I could remove this line instead of setting it to `true` again, but I like that it's explicit about what `CopyBytes` supports.